com/paypal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package com.paypal;
+
+public class Perfection {
```

Comment:
Please, create a unit test for each public method in this class to make sure this work according to the text.

Reply:
Also, improve the class name, so it explains better what does.